deploy/dev/devops-sample.yaml

```diff
@@ -28,7 +28,7 @@ spec:
               value: js|html
             - name: CACHE_PUBLIC_EXPIRATION
               value: 3d
-          image: $REGISTRY/$HARBOR_NAMESPACE/$APP_NAME:SNAPSHOT-$BRANCH_NAME-$BUILD_NUMBER
+          image: $REGISTRY/$DOCKERHUB_NAMESPACE/$APP_NAME:$BUILD_NUMBER
```

Comment:
Thanks for your pr.   

We use `SNAPSHOT-$BRANCH_NAME-` as the prefix while pushing the image. Please keep it.

https://github.com/kubesphere/devops-java-sample/blob/master/Jenkinsfile-online#L44

$REGISTRY/$DOCKERHUB_NAMESPACE/$APP_NAME:SNAPSHOT-$BRANCH_NAME-$BUILD_NUMBER